feature/common/etc/smartics/skipped-modules.xml

```diff
@@ -208,6 +212,10 @@
         <include artifact="io.sundr:sundr-codegen" />
     </module>
 
+    <module name="javax.transaction.jta" skip="true">
```

Comment:
Maybe best to keep the javax skipped dependencies in one place. E.g here:

https://github.com/wildfly-extras/wildfly-camel/blob/master/feature/common/etc/smartics/wildfly-modules.xml

Reply:
+1